### DISCUSSION TITLE
Actividad Docker - GRUPO 2

### DISCUSSION THREAD

**Discussion by BenjaminPeres596** [frlp-utn-ingsoft/catedra#113] [Show and tell]
https://github.com/JuaniLezcano/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones:
- archivo se llama “env-example” no lleva extencion, luego se edita y se lo renombra .env

Puntos aprobados:
* Construye la imagen
* buen tamaño de imagen
* despliega el contenedor

Salida de comandos:
$ docker images -a
vetsoft      latest    a0c78d9146e6   15 seconds ago   308MB
$ docker ps  -a
d4908a98a8bf   vetsoft   "python manage.py ru…"   8 seconds ago   Up 7 seconds   0.0.0.0:8000->80/tcp, :::8000->80/tcp   peaceful_austin

**Reply by ValentinMilocco** [reply]
Ya estan hechas las correcciones.
